puppetfactory/puppetfactory.gemspec

```diff
@@ -3,7 +3,7 @@ require 'date'
 
 Gem::Specification.new do |s|
   s.name              = "puppetfactory"
-  s.version           = '0.5.5'
+  s.version           = '0.5.6'
```

Comment:
@greglarkin It should be fine at this point since it's been merged to release. I think it's ok to do this when it's a PR to the release branch, to be immediately released.